src/ado2gh/Services/OrgsCsvGeneratorService.cs

```diff
@@ -1,33 +1,39 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace OctoshiftCLI.AdoToGithub
 {
     public class OrgsCsvGeneratorService
     {
-        public virtual async Task<string> Generate(
-            AdoApi ado,
-            IDictionary<string, IDictionary<string, IDictionary<string, IEnumerable<string>>>> pipelines)
+        private readonly AdoInspectorService _adoInspectorService;
+
+        public OrgsCsvGeneratorService(AdoInspectorService adoInspectorService)
+        {
+            _adoInspectorService = adoInspectorService;
+        }
+
+        public virtual async Task<string> Generate(AdoApi adoApi)
```

Comment:
`AdoApi` is a dependency and should be injected.

Reply:
leaving this as is, I'm now injecting an AdoInspectorServiceFactory, but we still need to the AdoApi object to call the factories create function.

Reply:
I'm not sure I agree, but I did make this change.